### ISSUE TITLE
Grafana dashboard does not displace correct data for farmer and pool addresses

### ISSUE THREAD

**Issue by sbates130272** [speedmann/prometheus-chia-exporter#6]
The prometheus exporter shows this:
```
# HELP chia_reward_address_info Farming rewards go to this address 
# TYPE chia_reward_address_info gauge
chia_reward_address_info{farmer_target="<redacted>",pool_target="<redacted>"} 1.0
```
but the grafana dashboard shows the 1.0 rather than the label values? I think this is an issue in the Grafana dashboard JSON?

**Comment by speedmann** [comment]
In grafana edit the panel and below the query you can add `{{ farmer_target }}` into the Legend field. For the pool address it is `{{ pool_target }}`. 
I will update the json file soon, i might have forgotten to save first before exporting... (Not a big fan of that save feature after all...)

**Comment by sbates130272** [comment]
@speedmann I had that but it did not work for me (it still showed 1). Can you confirm this worked for you.

**Comment by speedmann** [comment]
I just imported the dashboard and i get this
<img width="1601" alt="screeonshot" src="https://user-images.githubusercontent.com/424659/118191198-82ca3d80-b444-11eb-8cc6-207c89fc16a4.png">
What grafana version are you using? Maybe i used a feature which is not available in older versions.

Here it's grafana 7.5.5 (You can find the version in the lower left corner on the question mark)

**Comment by sbates130272** [comment]
@speedmann ah that was it. I was on 7.0. I upgraded my Grafana container and it works. Perhaps add a note on that requirement to the README?

**Comment by speedmann** [comment]
I updated the readme and included a note about grafana 7.5.5
